python/ray/tests/test_runtime_env_packaging.py

```diff
@@ -290,6 +290,7 @@ def handler(path):
     "parsing_tuple",
     [("gcs://file.zip", Protocol.GCS, "file.zip"),
      ("s3://bucket/file.zip", Protocol.S3, "s3_bucket_file.zip"),
+     ("http://test.com/file.zip", Protocol.HTTP, "https_test_com_file.zip"),
```

Comment:
The expected output for this test should be **http**_test_com_file.zip, not **https**_test_com_file.zip.